### ISSUE TITLE
AUC

### ISSUE THREAD

**Issue by gnujimmik** [ppwwyyxx/tensorpack#167]
Do you have plan to make AUC in stat? 

Best,

**Comment by ppwwyyxx** [comment]
Not currently. But you don't have to make something __into__ tensorpack to use it. 
It should be very easy to implement (just use `sklearn.metric.auc`).

**Comment by gnujimmik** [comment]
Thanks! I already made it.
